store/etcd/etcd.go

```diff
@@ -450,6 +453,7 @@ func (s *Etcd) WatchTreeExt(directory string, stopCh <-chan struct{}) (<-chan *s
 				LastIndex: result.Node.ModifiedIndex,
 				Action:    result.Action,
 				Dir:       result.Node.Dir,
+				Response:  result,
```

Comment:
it would be good to either export fields which are needed i.e add the fields which are missing form [this](https://github.com/coreos/etcd/blob/master/client/keys.go#L258) or remove all fields and return just response, either is valid, but doing partially is going to lead to confusion.